pom.xml

```diff
@@ -30,28 +30,28 @@
     </developers>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
-        <java.level>6</java.level>
+        <jenkins.version>1.642.1</jenkins.version>
```

Comment:
I'm trying to integrate the `@Symbol`, but I keep getting different errors. Either when using the Pipeline Syntax generator, or when creating/running jobs, etc.
What am I missing?
I know that the annotation comes from `org.jenkins-ci:symbol-annotation` dependency.
Is there any minimum version Jenkins core required?
Is there any mimimim `workflow-something` dependencies required? Or minimum version?

@jglick some extra hints would be useful.